lib/oai/QubitOai.class.php

```diff
@@ -294,16 +294,29 @@ public static function addOaiSet($oaiSet)
    * @param string $setSpec, the setSpec of an OAI set
    * @return QubitOaiSet/boolean the OAI set matched (or false if none matched)
    */
-  public static function getMatchingOaiSet($setSpec, $oaiSets)
+  public static function getMatchingOaiSet($setSpec)
   {
-    foreach ($oaiSets as $oaiSet)
+    // Check additional sets, if enabled
+    $setting = QubitSetting::getByName('oai_additional_sets_enabled');
+    $setsEnabled = (null === $setting) ? false : $setting->value;
+
+    if ($setsEnabled)
```

Comment:
It gets the source value without fallback, which is useful when you don't really care about translated content.